client/a8c-for-agencies/sections/sites/sites-dashboard/style.scss

```diff
@@ -1016,6 +1016,22 @@
 	display: none;
 }
 
+.sites-dashboard .a4a-site-performance {
+	.notice-banner {
+		display: flex;
+	}
+
+	.site-performance-device-tab__controls:not(.disabled) .segmented-control__item.is-selected {
+		 .segmented-control__link {
+			color: var(--color-text-white);
+		}
+
+		+ .segmented-control__item .segmented-control__link {
+			border-left-color: var(--color-link);
+		}
+	}
```

Comment:
We need to make some changes to the theme, which we will do later. For now, we will use these styles.

Reply:
Update: I'm planning the fix this in a different PR before we merge this so that we don't have to override.